packages/wrangler/src/miniflare-cli/assets.ts

```diff
@@ -29,7 +24,18 @@ export default async function generateASSETSBinding(options: Options) {
 			try {
 				const url = new URL(request.url);
 				url.host = `localhost:${options.proxyPort}`;
-				return await miniflareFetch(url, request);
+				const headers = new MiniflareHeaders(request.headers);
+				if (headers.get("Upgrade") === "websocket") {
+					headers.delete("Sec-Websocket-Accept");
+					headers.delete("Sec-Websocket-Key");
+				}
+				const proxyRequest = new MiniflareRequest(url, {
+					method: request.method,
+					headers,
+					cf: request.cf,
+					body: request.body,
+				});
```

Comment:
```suggestion
				const proxyRequest = new MiniflareRequest(url, request);
				if (proxyRequest.headers.get("Upgrade") === "websocket") {
					proxyRequest.headers.delete("Sec-WebSocket-Accept");
					proxyRequest.headers.delete("Sec-WebSocket-Key");
				}
```

Unlikely to ever matter, but there's a bunch of stuff that can be in a request. This makes sure we capture everything rather than just explicitly passing through a few properties.